### PR TITLE
HOCS-4029 reduce the sensitivity of the HPA and update the version of the main sidecar pods

### PR DESCRIPTION
**Pull request by nickmcmahon01**
This commit reduces the sensitivity of the HPA and updates the version of the main sidecar pods to latest.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -8,23 +8,28 @@ export VERSION=${VERSION}
 
 if [[ ${KUBE_NAMESPACE} == *prod ]]
 then
-    export CLUSTER_NAME="acp-prod"
     export CASE_CREATOR_UKVI_COMPLAINT_USER="TBD"
     export CASE_CREATOR_UKVI_COMPLAINT_TEAM="TBD"
     export CASE_CREATOR_UKVI_COMPLAINT_GROUP="TBD"
+
+    export MIN_REPLICAS="1"
```
Comment (LiamMacP) [inline deploy.sh:15]: If this is going to be the same for both, just pull it out the if block for now?

File: deploy.sh [hunk 2]
```diff
@@ -8,23 +8,28 @@ export VERSION=${VERSION}
...
 
-export KUBE_CERTIFICATE_AUTHORITY="https://raw.githubusercontent.com/UKHomeOffice/acp-ca/master/${CLUSTER_NAME}.crt"
-
-cd kd || exit 1
+cd kd
 
-kd --timeout 15m \
-    -f identityConfigMap.yaml \
-    -f deployment.yaml
+kd --insecure-skip-tls-verify \
```
Comment (LiamMacP) [inline deploy.sh:32]: We put the `KUBE_CERTIFICATE_AUTHORITY` stuff in previously, I don't think this is a correct change to ignore the certs?

File: kd/deployment.yaml [hunk 3]
```diff
@@ -8,12 +8,15 @@ metadata:
   annotations:
     downscaler/uptime: {{.UPTIME_PERIOD}}
 spec:
-  replicas: 1
-  strategy:
-    type: Recreate
+  replicas: {{.MIN_REPLICAS}}
```
Comment (LiamMacP) [inline kd/deployment.yaml:11]: Not using the `MAX_REPLICAS` anywhere?

**Review by LiamMacP — CHANGES_REQUESTED** [review]
Couple of changes/ questions here...

File: kd/deployment.yaml (hunk 3 above)
Comment (nickmcmahon01) [reply kd/deployment.yaml:11]: I want to add a HPA as part of the getting this service into live, i've coped this code in from another service.

File: deploy.sh (hunk 2 above)
Comment (nickmcmahon01) [reply deploy.sh:32]: 👍

File: deploy.sh (hunk 1 above)
Comment (nickmcmahon01) [reply deploy.sh:15]: I never trust that it will get put back in again..

Comment (LiamMacP) [reply deploy.sh:15]: Fair argument...

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**